src/applications/disability-benefits/2346/helpers/index.js

```diff
@@ -71,3 +72,20 @@ export function validateZIP(errors, zip) {
     );
   }
 }
+
+export const newAddressHider = formData => {
+  const address = get('newAddress', formData) || {};
```

Comment:
Instead of using our `get` module you could do `formData?.newAddress` if you'd like! 🙂

Reply:
Its also weird that there are 2 imports from utlities/data - Ill fix that too